packages/google-cloud-run/src/index.js

```diff
@@ -5,6 +5,16 @@
 
 'use strict';
 
+const minimumNodeJsVersion = require('@instana/core/src/util/minimumNodeJsVersion');
+if (require('@instana/core/src/util/nodeJsTooOld')(minimumNodeJsVersion)) {
```

Comment:
Suggestion: You could require the minimum node js version inside the `nodeJsTooOld` script (both live in core)
This will make the code a little bit more readable. And you can still offer to pass in a different node version

```
if (require('@instana/core/src/util/nodeJsTooOld')()) {
   ...
}
```

Reply:
Suggestion: You could even do

```
require('@instana/core/src/util/nodeJsTooOld')()
```

And put the console.error into the utility

Reply:
I have bundled both the minimum version and the check into one file, thereby we only have one new module and one line with imports now. I left the console.error in each entry file because the text is a bit different for each package.

Reply:
> require('@instana/core/src/util/nodeJsTooOld')()

That would actually be too minimal IMO, the important piece is to return early and stop evaluating the rest of the module.